glib-sys/src/lib.rs

```diff
@@ -1055,24 +1055,14 @@ impl ::std::fmt::Debug for GArray {
 }
 
 #[repr(C)]
-pub struct GAsyncQueue(c_void);
+pub struct _GAsyncQueue(c_void);
 
-impl ::std::fmt::Debug for GAsyncQueue {
-    fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
-        f.debug_struct(&format!("GAsyncQueue @ {:?}", self as *const _))
-         .finish()
-    }
-}
+pub type GAsyncQueue = *mut _GAsyncQueue;
 
 #[repr(C)]
-pub struct GBookmarkFile(c_void);
+pub struct _GBookmarkFile(c_void);
 
-impl ::std::fmt::Debug for GBookmarkFile {
-    fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
-        f.debug_struct(&format!("GBookmarkFile @ {:?}", self as *const _))
-         .finish()
-    }
-}
+pub type GBookmarkFile = *mut _GBookmarkFile;
```

Comment:
Why this representation change?

Reply:
This current representation for disguised types,
made for GdkAtom and GIConv

Reply:
Good for you @sdroege ?

Reply:
Yes, was just curious if this was intentional and why.